data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,281 @@ mission "Arachi Cartoonist 2"
+				goto choice
+			
+			
+			label arachi
+			apply
+				set "coalition games: arachi"
+			`	When the train stops, you find yourself inside a building similar to a spaceport, but the ships are more akin to a mix between a helicopter and a raft. As other visitors start boarding the flying platforms, you follow suit and find that each one is filled by dozens of adjustable seats. It seems these seats' default position is designed for Arachi, but moving a few parts makes them capable of easily accommodating Kimek and even Saryds. After fiddling with yours for a while, you manage to adjust it to a configuration which is suitable enough, inasmuch as it is more comfortable than the others.`
+			`	Once the rafts are full, they leave their docks and depart the port. Leaving the building, you realize it was built into the side of a highland, and you're immediately soaring out over a lush rainforest. Increasingly larger trees emerge from the carpet of green below, which seems to shimmer with the movement of the various animals in the canopy. Bird-like creatures with funneled beaks, startled by the rafts' engines, appear in large flocks and soar ahead of and away from the fleet of flying rafts before settling back into the trees. In the distance, there is a river so wide you at first believe it to be an ocean, until a Heliarch agent informs you otherwise.`
+			`	You approach a cluster of nine colossal trees which stand out from the rest. One in the center is taller than all the others, and the other eight are varying heights arranged regularly around it. Smaller rafts are also approaching the site from elsewhere, each with a different circular symbol on it. The Heliarch agents on your own raft are busy preparing to land, so you ask a Saryd seated next to you what they mean after noticing she's carrying a translation box.`
+			`	"The multiple Arach houses, the symbols represent," she says. "To be patrons of athletes, a custom of members of the great houses it is."`
+			`	The rafts near the center of the tree cluster and begin to lower. Up close, you can make out details you couldn't before. The trees seem similar to oak trees but have leaves several times larger. Vines hang from the branches, many of which extend to other trees. As you get even closer, you realize most are not actually vines, but rather ropes which connect the trees in an intricate, scattered web. Dozens of medium-sized rafts gather in groups under the ropes and attach something between themselves. After a few minutes, they move away, revealing that they are now supporting large nets below each rope. Finally, a raft with multiple large monitors orients itself above everything and displays the symbols of the Arachi houses you noticed on the rafts earlier followed by a list of the participants from each house.`
+			`	"Damaged, the forest would be, if built here, grand colosseums were," the Saryd explains. "Carefully cultivated, this area was, so that for the games, used it can be."`
```

Comment:
```suggestion
			`	"Damaged, the forest would be, if built here, grand coliseums were," the Saryd explains. "Carefully cultivated, this area was, so that for the games, used it can be."`
```
I think in English "Colosseum" is reserved for the Italian monument, while "coliseum" is used to refer to a large stadium.
Although here it's used by an alien translation device, so it's not that important.